egs2/chime7_task1/asr1/README.md

```diff
@@ -278,10 +307,51 @@ by computing SA-WER for each speaker, where the hypothesis are re-ordered based
 2. `ERROR: Could not install packages due to an OSError: [Errno 2] No such file or directory: 'YOUR_PATH/espnet/tools/venv/lib/pyth
 on3.9/site-packages/numpy-1.23.5.dist-info/METADATA'`. This is due to numpy installation getting corrupted for some reason.
 You can remove the site-packages/numpy- folder manually and try to reinstall numpy 1.23.5 with pip.
+3. `FileNotFoundError: [Errno 2] No such file or directory: 'PATH2YOURESPNET/espnet/tools/venv/bin/sox'
+` during CHiME-6 generation from CHiME-5, `correct_signals_for_clock_drift.py` script: try to install conda sox again, via `conda install -c conda-forge sox`.
+4. `ModuleNotFoundError: No module named 's3prl'` for some reason s3prl did not install, run `YOUR_ESPNET_ROOT/tools/installers/install_s3prl.sh`
+5. `Command 'gss' not found` for some reason gss did not install, you can run `YOUR_ESPNET_ROOT/tools/installers/install_gss.sh`
+7. `wav-reverberate command not found` you need to install Kaldi. go to `YOUR_ESPNET_ROOT/tools/kaldi` and follow the instructions
+in `INSTALL`.
+8. `WARNING  [enhancer.py:245] Out of memory error while processing the batch` you got out-of-memory (OOM) when running GSS.
+You could try changing parameters as `gss_max_batch_dur` and in local/run_gss.sh `context-duration`
+(this latter could degrade results however). See local/run_gss.sh for more info.
+9. Much worse WER than baseline and you are using `run.pl`. Check the GSS results, GSS currently does not work well
+if you use multi-gpu inference and your GPUs are in shared mode. You need to run `set nvidia-smi -c 3`.
+
+
+## Memory Consumption (Useful for SLURM etc.)
+
+Figures kindly reported by Christoph Boeddeker, running this baseline code
+on Paderborn Center for Parallel Computing cluster (which uses SLURM).
+These figures could be useful to anyone that uses job schedulers and clusters
+for which resources are assigned strictly (e.g. job killed if it exceed requested
+memory resources).
+
+Used as default:
+ - train: 3G mem
+ - cuda: 4G mem (1 GPU)
+ - decode: 4G mem
+
+GSS:
+ - nj=8 (default 4)
+ - time 45 h per job (but might be faster)
+ - num_threads=5 (1 for main process, 4 for DataLoader processes)
+ - Mem: 2 GB per core or 10 GB per job. In SLURM it is --mem 2GB, since mem means mem-per-core in SLURM.
+
+scripts/audio/format_wav_scp.sh:
+ - Some spikes to the range of 15 to 17 GB
+
+${python} -m espnet2.bin.${asr_task}_inference${inference_bin_tag}:
```

Comment:
Oh, the github markdown supports latex syntax.  I didn't know it

${python} -m espnet2.bin.${asr_task}_inference${inference_bin_tag}:

Reply:
Me neither ! that is actually very cool